.editorconfig

```diff
@@ -14,8 +14,5 @@ trim_trailing_whitespace = false
 [*.{yml,yaml,json}]
 indent_size = 2
 
-[composer.{json,lock}]
-indent_size = 4
-
-[auth.json]
+[{composer, auth}.json]
```

Comment:
@engcom-Foxtrot basically it's not the same. You just removed option for `composer.lock` files.

Reply:
@ihor-sviziev yes, composer.lock inherits settings from general configuration (*).